chapter09/dtj/ex3

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+## Copyright (C) 2015 by dtj
+
+use warnings;
+use strict;
+
+if ( !$ARGV[0] || !open INPUT, "<", $ARGV[0] ) {
+  die "Could not open file for reading\n";
+}
+
+open OUTPUT, ">", $ARGV[0] . ".out";
+
+# Use something random, as it's less likely to appear in the input
```

Comment:
https://xkcd.com/221/

I wanted something that was less likely to be in the text already as the temporary thing.